tests/integration/test_issue_comment.py

```diff
@@ -57,6 +57,11 @@ def mock_comment(request):
         .and_return(True)
     )
     flexmock(project_class).should_receive("issue_comment").and_return(None)
+    issue = flexmock()
+    flexmock(project_class).should_receive("get_issue").and_return(issue)
+    comment = flexmock()
+    flexmock(issue).should_receive("get_comment").and_return(comment)
+    flexmock(comment).should_receive("add_reaction").with_args("+1").once()
```

Comment:
Just out of interest, what happens here with both prod and stg deployed?

I hope it doesn't fail [here](https://github.com/packit/packit-service/pull/1222/files#diff-1d719ae684f53243de1efeadb4b79e8cd77b2408c27b563bd43e0c4b6530236aR100)

Reply:
Now that I'm thinking about it, I hope GitHub apps can react :thinking: but i think i've seen some „voting“ mechanism implemented via bot, so it should be fine.

Reply:
I guess we'll find out once this is deployed to stg :D

Reply:
definitely a good point though, I hope we won't need to request a new perm for this